packages/doctor_pet/lib/core/data/mypatients.dart

```diff
@@ -0,0 +1,17 @@
+class Mypatients {
```

Comment:
Change into MyPatients or Patients or Patient. I think you should use Patient because I just only 1 patient represents 1 model